src/wallet/rpc/spend.cpp

```diff
@@ -1241,6 +1248,11 @@ RPCHelpMan send()
             // be overridden by options.add_inputs.
             coin_control.m_add_inputs = rawTx.vin.size() == 0;
             SetOptionsInputWeights(options["inputs"], options);
+
+            if (options.exists("avoid_change")) {
```

Comment:
I think this can be removed from send() because it is already run in FundTransaction().

Reply:
Hmm, interesting! Gonna check it